src/VirtoCommerce.Platform.Web/wwwroot/Localizations/en.VirtoCommerce.Platform.json

```diff
@@ -446,6 +436,46 @@
                     "name": "Name"
                 }
             },
+            "oauthapps": {
+                "labels": {
+                    "clientId": "Client id",
+                    "displayName": "Display name",
+                    "type": "Type",
+                    "clientSecret": "Client secret",
+                    "assigned-permissions": "Assigned permissions",
+                    "redirectUris": "Redirect uris",
+                    "postLogoutRedirectUris": "Post logout redirect uris"
+                },
+                "placeholders": {
+                    "clientId": "Input client id",
+                    "displayName": "Input display name",
+                    "type": "Input type",
+                    "clientSecret": "Input client secret"
+                }
+            },
+            "oauthapps-list": {
+                "title": "OAuth applications",
+                "subtitle": "Security service"
+            },
+            "oauthapps-detail": {
+                "title": "Edit OAuth application",
+                "title-new": "Create OAuth application",
+                "subtitle": "Security service",
+                "labels": {
+                    "clientSecret-warning": "Warning, copy client secret befor close this blade",
```

Comment:
"Warning! You must copy the secret now. It will be hidden after closing the blade."

Reply:
done